src/client/index.js

```diff
@@ -390,6 +391,17 @@ function mkcircuit(data, opts) {
     monitorview.on('change:pixelsPerTick', show_scale);
 }
 
+function exportCircuitAsSvg() {
+    // Create an artificial link and click it to download the temporary file
+    let downloadLink = document.createElement('a');
+    downloadLink.download = 'exported_circuit.svg';
+    downloadLink.href = circuit.getSvgExportUrl(document.querySelector('#paper svg'));
```

Comment:
The API is not clean. It would be better if the API accepted e.g. a paper to be saved as SVG. Also, the current implementation of the `getSvgExportUrl` method modifies the SVG element of the paper, which (I think) makes it lose functionality. This should be avoided.